helm/cloudnative-pg/templates/cilium-network-policy.yaml

```diff
@@ -0,0 +1,31 @@
+{{ if .Values.ciliumNetworkPolicy.enabled }}
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ include "cloudnative-pg.fullname" . }}-cnp
+  labels:
+    {{- include "cloudnative-pg.labels" . | nindent 4 }}
+spec:
+  endpointSelector:
+    matchLabels:
+       {{- include "cloudnative-pg.selectorLabels" . | nindent 6 }}
+  egress:
+    # Allow egress to Kubernetes API server.
+    - toEntities:
+        - kube-apiserver
+    # Allow DNS.
```

Comment:
Just for curiosity: What does it need DNS for?

Reply:
Turns out, nothing! I removed it